webapp/channels/src/components/confirm_modal.tsx

```diff
@@ -166,46 +167,52 @@ export default class ConfirmModal extends React.Component<Props, State> {
             );
         }
 
+        const headerText = (
+            <div className='ConfirmModal__header'>
```

Comment:
Why are these extra elements needed here? I would hope that we could just pass `this.props.title` directly as `modalHeaderText` below